kratos/sources/model_part_io.cpp

```diff
@@ -4423,7 +4485,7 @@ namespace Kratos
     // matrix value is completely read. It can be used to read any kind of
     // vector or matrix with operator >> defined and writtern in following
     // format for a vector: [size] ( value1, value2,...., valueN )
-    // format for a matrix: [size1,size2] ( )( )...( ) //look props read
+    // format for a matrix: [size1,size2] (( )( )...( )) //look props read
```

Comment:
Doxygen can read this?

Reply:
Doxygen? At least it works in the test..

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/36217018-a5c2a5e6-11b0-11e8-8a3a-7ea45c5d9040.png)

I meant this

Reply:
I'm sorry but I have absolutely no idea what you are talking about -.-

Reply:
Doxygen is a documentation generator that uses the comments in the code to build a documentation

https://www.stack.nl/~dimitri/doxygen/

Our idea is to start supporting Doxygen and generate the Kratos documentation soon, for that reason we need to start being serious about requesting documented methods and classes, or it will be more difficult to solve this in the future